### PR TITLE
Update documentation

### LINKED ISSUE #68: Update User Guide
- [x] Fatin: 1, 2
- [x] YiJing: 3
- [x] Sam: 4.1 - 4.2
- [x] Gabe: 4.3
- [x] Rai: 4.4 - 4.7

### LINKED ISSUE #69: Update Developer Guide
- [x] A: YiJing
- [x] B: Rai
- [x] C: Gabe, Fatin
- [x] D, E: Sam

### PR DESCRIPTION
**Pull request by PotatoCombat**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -308,40 +308,224 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
...
+Use case resumes at step 2.
+
+=== Use case: Enroll student into class
+
+*MSS*
+
+1. User requests to enroll a student in a session group.
+2. TA-Tracker registers the student in the session group.
+3. TA-Tracker switches to the Student View.
+4. TA-Tracker shows the student in the student list for the session group.
```
Comment (aakanksha-rai) [inline docs/DeveloperGuide.adoc:778]: There should be more 'ping-ponging' in the use cases.

**Review by fatin99 — APPROVED** [review]
all good :)

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (PotatoCombat) [reply docs/DeveloperGuide.adoc:778]: Ok. Shall we rephrase the use cases in a separate branch?

Comment (aakanksha-rai) [reply docs/DeveloperGuide.adoc:778]: Sounds good.

**Pull request merged**